src/cascadia/TerminalApp/defaults.json

```diff
@@ -0,0 +1,216 @@
+            { "command": "switchToTab1", "keys": ["ctrl+alt+2"] },
+            { "command": "switchToTab2", "keys": ["ctrl+alt+3"] },
+            { "command": "switchToTab3", "keys": ["ctrl+alt+4"] },
+            { "command": "switchToTab4", "keys": ["ctrl+alt+5"] },
+            { "command": "switchToTab5", "keys": ["ctrl+alt+6"] },
+            { "command": "switchToTab6", "keys": ["ctrl+alt+7"] },
+            { "command": "switchToTab7", "keys": ["ctrl+alt+8"] },
+            { "command": "switchToTab8", "keys": ["ctrl+alt+9"] }
+        ],
+        "requestedTheme": "system",
+        "showTabsInTitlebar": true,
+        "showTerminalTitleInTitlebar": true,
+        "wordDelimiters": " ./\\()\"'-:,.;<>~!@#$%^&*|+=[]{}~?\u2502"
+    },
+    "profiles":
+    [
+        {
+            "guid": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
+            "name": "Windows PowerShell",
+            "commandline": "powershell.exe",
+            "startingDirectory": "%USERPROFILE%",
+            "background": "#012456",
+            "closeOnExit": true,
+            "colorScheme": "Campbell",
+            "cursorColor": "#FFFFFF",
+            "cursorShape": "bar",
+            "fontFace": "Consolas",
+            "fontSize": 10,
+            "historySize": 9001,
+            "icon": "ms-appx:///ProfileIcons/{61c54bbd-c2c6-5271-96e7-009a87ff44bf}.png",
+            "padding": "0, 0, 0, 0",
+            "snapOnInput": true,
+            "useAcrylic": false
+        },
+        {
+            "guid": "{0caa0dad-35be-5f56-a8ff-afceeeaa6101}",
+            "name": "cmd",
+            "commandline": "cmd.exe",
+            "startingDirectory": "%USERPROFILE%",
+            "closeOnExit": true,
+            "colorScheme": "Campbell",
+            "cursorColor": "#FFFFFF",
+            "cursorShape": "bar",
+            "fontFace": "Consolas",
+            "fontSize": 10,
+            "historySize": 9001,
+            "icon": "ms-appx:///ProfileIcons/{0caa0dad-35be-5f56-a8ff-afceeeaa6101}.png",
+            "padding": "0, 0, 0, 0",
+            "snapOnInput": true,
+            "useAcrylic": true,
+            "acrylicOpacity": 0.75
+        }
+    ],
+    "schemes":
+    [
+        {
+            "name": "Campbell",
+            "foreground": "#CCCCCC",
+            "background": "#0C0C0C",
+            "black": "#0C0C0C",
+            "red": "#C50F1F",
+            "green": "#13A10E",
+            "yellow": "#C19C00",
+            "blue": "#0037DA",
+            "purple": "#881798",
+            "cyan": "#3A96DD",
+            "white": "#CCCCCC",
+            "brightBlack": "#767676",
+            "brightRed": "#E74856",
+            "brightGreen": "#16C60C",
+            "brightYellow": "#F9F1A5",
+            "brightBlue": "#3B78FF",
+            "brightPurple": "#B4009E",
+            "brightCyan": "#61D6D6",
+            "brightWhite": "#F2F2F2"
+        },
+        {
+            "name": "Vintage",
+            "foreground": "#C0C0C0",

```

Comment:
can we delete all of their generator code?

Reply:
omg YES WE CAN